<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>313</width>
    <height>223</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>252</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="totalBalance">
       <property name="text">
        <string>Total balance:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="totalBalanceLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Operations</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="depositRadioButton">
          <property name="text">
           <string>Deposit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="withdrawRadioButton">
          <property name="text">
           <string>Withdraw</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="amount">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="amountLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="operationPushButton">
       <property name="text">
        <string>Do</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
